### PR TITLE
Setup File

### PR DESCRIPTION
**Pull request by BabarKhanDev**
Add script to download and extract the COCO dataset into Datasets/coco/

Only tested on macOS but I see no reason why it won't work on Windows or Linux

### CODE REVIEW

**Comment by Laura7089** [comment]
Could probably use a shell shebang at the top:

```bash
!#/bin/sh
set -eux
```

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#COCO Dataset
+mkdir Datasets
+cd Datasets
+mkdir coco
+cd coco
+mkdir images
+cd images
```
Comment (markspolakovs) [inline setup.sh]: Nitpick/suggestion:

```suggestion
mkdir -p Datasets/coco/images
cd Datasets/coco/images
```

File: setup.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
+!#/bin/sh
```
Comment (Laura7089) [inline setup.sh]: My bad, this should be `#!/bin/sh`.

**Review by Laura7089 — CHANGES_REQUESTED** [review]
Instead of duplicating the commands 4 times for each dataset download, you could use a for loop + array of URLs and/or a function to download and extract them.

It will work without (so long as you fix the shebang issue).

**Review by markspolakovs — APPROVED** [review]
Untested but code LGTM

**Review by Laura7089 — APPROVED** [review]

**Pull request merged**